src/inverted-index.js

```diff
@@ -1,88 +1,143 @@
-function getTokens(text){
-	//remove punctuations and other characters
-	let cleanText = text.replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g,"")
-	                    //remove excess whitespace
-	                    .replace(/\s{2,}/g," ")
-	                    //convert to lower case
-	                    .toLowerCase()
-	                    //split into words
-	                    .split(' ')
-	                    //sort
-	                    .sort();
-	return(cleanText);
-}
-
 class InvertedIndex{
 	constructor(){
-		//Object to store the index
-		this.index = {};
+		//object to store all created indexes
+		this.indexes = {};
 	}
 
 	/*
-	* Function to separate a string into words
-	* Non-alphanumeric characters are removed 
-	* The text is converted to lower case and sorted
-	*/
-	static clean(text){
-		let cleanText = text.replace(/[.,\/#!$%\^&\*;:{}=\-_`~()]/g,"")
-		 					.replace(/\s{2,}/g, " ")
-		 					.toLowerCase()
-		 					.split(' ')
-		 					.sort();
+     * @param{String} text - Words to get tokens from
+     * @preturn{Array} cleanText - An array of sorted strings without non-alphanumeric symbols
+     **/
+	clean(text){
+		let cleanText = text.replace(/[\[\].,\/#!$%\^&\*;:{}?=\-_`~()]/g,"")
+		 					          .replace(/\s{2,}/g, " ")
+		 					          .toLowerCase()
+		 					          .split(' ')
+		 					          .sort();
 		return(cleanText);
 	}
 
 	/*
-	* Function to remove duplicate words
-	* from an array of strings
+	* @param{Array} words - An array of strings
+	* @param{Array} uniqueWords - An array non-duplicate strings
 	*/
 
-	static removeDuplicates(words){
+	removeDuplicates(words){
 		let uniqueWords = words.filter((item, index) => words.indexOf(item) === index);
 		return(uniqueWords);
 	}
 
-	getIndex(docToIndex){
-		const index = {};
+	//Efizi function to sort the Object keys
+	sortObjectKeys(index){
+		let sortedKeys = Object.keys(index).sort();
+		let sortedObject = {}; //Object that will contain the sorted object
+		sortedKeys.forEach((key)=>{
+			sortedObject[key] = index[key];
+		});
+		return(sortedObject);
+	}
 
-		docToIndex.forEach((document, docIndex) => {
-			let cleanWords = InvertedIndex.clean(document.text);
-			//remove duplicates from words
-			let uniqueWords = InvertedIndex.removeDuplicates(cleanWords);
+	/*
+	* @param{String} filename - Name of the file for which index is to be created
+	* This function does not return anything. It simply updates the indexes object
+	**/
+createIndex(filename, docToIndex){
```

Comment:
It's a good practice to always have a space between these two ) { in all your code blocks.
So this 
`createIndex(filename, docToIndex){`
Should be this
`createIndex(filename, docToIndex) {`